packages/yourloops/components/patient-list/current-patient-list/current-patient-list.hook.tsx

```diff
@@ -193,11 +192,8 @@ export const useCurrentPatientListHook = (props: CurrentPatientListProps): Curre
       {
         type: 'number',
         field: PatientListColumns.TimeInRange,
-        headerName: t('time-in-range'),
-        renderHeader: () => <CustomHeaderWithTooltip
-          tooltipText={t('Time In Range')}
-          headerTitle={t('time-in-range')}
-        />,
+        headerName: t('column-header-time-in-range'),
+        description: t('time-in-range'),
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏The tooltip of the column "TIR" used to be "Time In Range" and now it's "Time in range", which is wrong

On this branch:
![image](https://github.com/mdblp/blip/assets/116577202/4e14b6d8-228d-42ea-b444-74394b0851ed)

On Sandbox:
![image](https://github.com/mdblp/blip/assets/116577202/70455848-7f47-4cb1-9bfe-e0582d6629de)